app/amp_start.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python2.7
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from start import Handler
+
+
+class AMPstartHandler(Handler):
+    repo_required = False
+
+    def get(self):
+        self.render('start.html', cache_seconds=0, get_vars=self.get_vars)
```

Comment:
There is very small difference I re-write more simply.

Reply:
Call:
```python
super(Handler, self).get()
```
instead of this line if there is no problem executing:
```python
self.env.robots_ok = True
```
in this class. This way, you can avoid writing similar line twice.